catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -390,6 +395,69 @@ public void testExecuteQueryCacheDisabled() throws Exception {
                 .size(), is(0));
     }
 
+    @Test
+    public void testMaxQueryPaging() throws Exception {
```

Comment:
We reach the total number of hits for many of the pre-existing tests. We reach the hard-coded limit in this test (1100 total hits with simulated max catalog result of 600 and 1000 results returned over two pages). These two conditions cover not using paging and paging. I can add more tests, but I might be missing what you mean by the number requested.